api/TimeAddressableMediaStore.yaml

```diff
@@ -1937,6 +1999,18 @@ paths:
           required: true
           schema:
             type: string
+        - name: flow_tag.{name}
+          in: query
+          description: |
+            Filter `referenced_by_flows` on tag values. This option is the same as the `tag.{name}` query parameter on the `/flows/` API endpoint.
```

Comment:
My gut feeling is that this should also "filter" the flow id from the first_referenced_by_flow field in the same way. Otherwise if we are using this tag query parameter to provide an early implementation of RBAC/ABAC we would be removing a flow id from `referenced_by_flows` only to leave it visible in `first_referenced_by_fMy gut feeling is that this should also "filter" the flow id from the first_referenced_by_flow field in the same way. Otherwise if we are using this tag query parameter to provide an early implementation of RBAC/ABAC we would be removing a flow id from `referenced_by_flows` only to leave it visible in `first_referenced_by_flow`. Just a thought and i'm not "wedded to it".low`. Just a thought and i'm not "wedded to it".

Reply:
I think you're right. But I think what we choose to do here will be affected by the discussion on whether we are happy to elevate auth classes strait to the core spec. So I'll hold off on this change for now.